### DISCUSSION TITLE
Help me!!!! Porque no me reconoce la variable pokeInput

### DISCUSSION THREAD

**Discussion by jaaramirez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#460] [Q&A]
Hola buenas noches, podrían ayudarme con esta duda, ya me atore se supone que segui los pasos para armar la pokedex pero no me reconoce la función de pokeInput y debido a esto no puedo llamar  a la busqueda de pokemon en la Api, revise el codigo pero no se en donde me estoy equivacando, les agradecere mucho sus valiosas observaciones.

![image](https://user-images.githubusercontent.com/89113801/159106133-c51fae3d-dafb-4f97-babc-32ac87fb5d12.png)

**Comment by JavierMendez-Coder** [comment]
Buenas noches. Para utilizar un **Template Literal** tienes que poner el texto entre acentos graves ( ` ) y no entre comillas ( ' o " ).
```javascript
const url = `https://pokeapi.co/api/v2/pokemon/${pokeInput}`;
```
Espero haberte ayudado. 👋

**Comment by Ingrzs** [comment]
esa comilla que tienes no es la correcta , tu tienes esta ' y es esta `
